spanish/wg-common/focus-areas/contributions/README.md

```diff
@@ -1,11 +1,9 @@
 ## Contributions
 
-**Goal:** Identify what contributions are made (e.g., code commit, wiki edit, documentation)
+**Goal:** Idntificar los tipos de contribuciones que se realizan (por ejemplo, commit de los códitos, ediciones en wiki, documentación) 
```

Comment:
**Meta:** Indentificar qué tipos de contibuciones se realizan (... commit del código...

Reply:
instead of "commit del código" let's use "comentario" since it's the name that GitHub in spanish uses :)